repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTScanner.java

```diff
@@ -45,7 +45,8 @@
 
     private final InspectBuilds inspectBuilds;
     private final InspectJobs inspectJobs;
-    private final DockerPipelineRunner pipelineRunner;
+    private final DockerPipelineRunner DockerPipelineRunner;
```

Comment:
same comment as below.

Since there is a lot of code repetition between both Launcher class, which hinders evolution, we may consider a refactoring to share the common code. (in another PR of course, before or after this one :-)